### ISSUE TITLE
Repository update fails already if only one repository is offline

### ISSUE THREAD

**Issue by tobias-klein** [ezra-bible-app/ezra-bible-app#395]
There are eight repositories in CrossWire's master repository list. If any of them fail to update the complete update fails. The behavior should be changed:

- Succeed if two or less repositories fail to update
- Fail if at least two repositories fail to update

**Comment by zhuiks** [comment]
This give us some trouble running automated acceptance tests!

@tobias-klein from Node-Sword-Interface side can you provide more details as a result from `updateRepositoryConfig()`?

We can make decision then in front-end, based on the repository's online statuses

**Comment by tobias-klein** [comment]
Yes, you're right @zhuiks
I was already thinking about this one when seeing the GitHub actions issues this week.

And yes, we could return a more detailed status from updateRepositoryConfig. It could be an Object with a status attribute for each repository that is queried.

**Comment by tobias-klein** [comment]
@zhuiks 
I have just published node-sword-interface 0.238.0 (not integrated here yet - feel free to take that up).

In the new version of node-sword-interface the function updateRepositoryConfig returns an object like this:

```
{
  result: false,
  'Bible.org': false,
  CrossWire: true,
  'CrossWire Attic': true,
  'CrossWire Beta': true,
  'CrossWire Wycliffe': true,
  'Deutsche Bibelgesellschaft': true,
  IBT: true,
  'Lockman Foundation': true,
  Xiphos: true,
  'eBible.org': true
}
```
The status value is simply a boolean. `true` means successful and `false` that the update failed for that particular repo. In the example above you see that the Bible.org repo is currently failing to update correctly.
